jest.config.js

```diff
@@ -1,10 +1,16 @@
 export default {
-  testEnvironment: "node",
+  testEnvironment: "jsdom",
```

Comment:
This change is so I can actually perform client-based testing (otherwise there is no `document`)